partcad-cli/src/partcad_cli/click/commands/list/assemblies.py

```
@@ -0,0 +1,72 @@
+import rich_click as click
+from partcad import logging
+from partcad.logging import Process
+
+
+@click.command(help="List available assemblies")
+@click.option("-r", "--recursive", is_flag=True, help="Recursively process all imported packages")
+@click.option(
+    "-u", "--used_by", type=str, required=False, help="Only process objects used by the given assembly or scene."
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)